### ISSUE TITLE
Don't lock Flush options to Prime tower

### ISSUE THREAD

**Issue by domunus** [bambulab/BambuStudio#948]
**Is your feature request related to a problem? Please describe.**
In the most recent version 1.4.0.18 you guys decided to lock "Flush options" to "Prime tower" being enabled.   Please don't lock features to the prime tower option.  It's a decent feature, but completely unnecessary with other proper settings it just creates unnecessary filament waste.  I've done quite a few multi-color prints without a prime towers with no issues.   With the proper Flushing volumes and the use of both flush options multi-color prints have been coming out great. 

This should be a decision by the user.  If they don't want to take the time to tweak settings then they can enable prime tower and purge a ton of filaments to be on the safe side.  But it shouldn't be a forced setting.  

**Describe the solution you'd like**
Remove the Prime tower requirement when wanting to use the Flush options.

**Comment by luis-godinez** [comment]
I totally agree. It would be great if we use the flushing for infill printing without the need for a prime tower. This would save a significant amount of time. I understand that this would only work if you weren't using opaque or transparent colors to avoid seeing the rainbow infill but as @domunus mentioned, this would be something in control of the end user.

**Comment by flodesi** [comment]
+1 on this. Would really appreciate this being implemented. At the very least have "Flush options" and "Prime tower" not bound to one another upon toggling advanced options. This would be the best of both worlds for more beginner users and those of us who know when it is proper to have one independent of the other.

**Comment by CristianGTR97** [comment]
It's funny that this still hasn't been solved
